nodejs/eks/cni-addon.ts

```diff
@@ -0,0 +1,487 @@
+// Copyright 2016-2014, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+import * as k8s from "@pulumi/kubernetes";
+
+export interface VpcCniAddonOptions {
```

Comment:
That's a 1:1 copy of the existing VpcCni addons except for the `enableIPv6` setting that is not automatically handled by the addon.